### PR TITLE
Issue1122 air exchange boolean

### PR DESCRIPTION
**Pull request by larissakuehn**
Issue #1122 is solved by replacing use_AirExchange by two booleans: 
- use_NaturalAirExchange
- use_MechanicalAirExchange

### CODE REVIEW

File: AixLib/ThermalZones/ReducedOrder/Multizone/BaseClasses/PartialMultizone.mo [hunk 1]
```diff
@@ -18,8 +18,10 @@ partial model PartialMultizone "Partial model for multizone models"
     "Number of fluid ports"
     annotation(Evaluate=true,
     Dialog(connectorSizing=true, tab="General",group="Ports"));
-  parameter Boolean use_AirExchange=true
-    "Consider infiltration and ventilation by setting true";
+  parameter Boolean use_NaturalAirExchange=use_MechanicalAirExchange
+    "Consider natural infiltration by setting true";
+  parameter Boolean use_MechanicalAirExchange=true
+    "Consider mechanical ventilation infiltration by setting true";
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/Multizone/BaseClasses/PartialMultizone.mo:24]: Remove "infiltration"

File: AixLib/package.mo [hunk 2]
```diff
@@ -80,8 +80,9 @@ package AixLib
     version="1.0.1", 
                       script="modelica://AixLib/Resources/Scripts/ConvertAixLib_from_1.0.1_to_1.0.2.mos",
 	version="1.0.2", 
-                      script="modelica://AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos")),
-
+                      script="modelica://AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos",
+    version= "1.0.3",
```
Comment (FWuellhorst) [inline AixLib/package.mo:84]: New version 1.0.4 is missing in the version specifier above these conversion files

File: AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo [hunk 3]
```diff
@@ -1,15 +1,19 @@
-﻿within AixLib.ThermalZones.ReducedOrder.ThermalZone;
+within AixLib.ThermalZones.ReducedOrder.ThermalZone;
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo:1]: Is the style check currently skipped? It seems like this file has a lot of style related changes

File: AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
 model ThermalZone "Thermal zone containing moisture balance"
-  extends AixLib.ThermalZones.ReducedOrder.ThermalZone.BaseClasses.PartialThermalZone;
+  extends
+    AixLib.ThermalZones.ReducedOrder.ThermalZone.BaseClasses.PartialThermalZone;
 
   replaceable model corG = SolarGain.CorrectionGDoublePane
     constrainedby AixLib.ThermalZones.ReducedOrder.SolarGain.BaseClasses.PartialCorrectionG
+    AixLib.ThermalZones.ReducedOrder.SolarGain.BaseClasses.PartialCorrectionG
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo:8]: remove

File: AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
     "Model for correction of solar transmission"
     annotation(choicesAllMatching=true);
   parameter Integer internalGainsMode = 1
      "decides which internal gains model for persons is used";
-  parameter Boolean use_AirExchange = false
-    "Consider infiltration and ventilation by setting true";
+  parameter Boolean use_NaturalAirExchange = use_MechanicalAirExchange
+    "Consider natural infiltration and ventilation by setting true";
+ parameter Boolean use_MechanicalAirExchange = false
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo:15]: indent

File: AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo [hunk 6]
```diff
@@ -688,50 +762,50 @@ equation
...
-                                                 color={191,0,0}));
-  connect(airExcMoi.ventRate, addInfVen.y) annotation (Line(points={{-21.2,
-          -9.12},{-24,-9.12},{-24,-28},{-27.4,-28}}, color={0,0,127}));
-  connect(airExcMoi.port_b, ROM.intGainsConv) annotation (Line(points={{-6,-4},
-          {58,-4},{58,78},{86,78}}, color={191,0,0}));
-  connect(airExcMoi.QLat_flow, SumQLat2_flow.u[3]) annotation (Line(points={{
-          -5.68,-8.96},{-6,-8.96},{-6,-40},{-42,-40},{-42,-64.8},{-40,-64.8}},
+  connect(airExcMoi.port_a, preTemVen.port)    annotation (Line(points={{-22,-6},
+          {-26,-6},{-26,-1},{-32,-1}},           color={191,0,0}));
+// connect(airExcMoi.ventRate, addInfVen.y) annotation (Line(points={{-21.2,
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo:767]: remove if not required

File: AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo [hunk 7]
```diff
@@ -744,6 +818,7 @@ equation
           -40},{110,-40}},                   color={0,0,127}));
   connect(QIntGainsInternalTot_flow.y, QIntGains_flow) annotation (Line(points={{98.2,
           -40},{110,-40}},                   color={0,0,127}));
+>>>>>>>>> Temporary merge branch 2
```
Comment (FWuellhorst) [inline AixLib/ThermalZones/ReducedOrder/ThermalZone/ThermalZone.mo:821]: remove

**Review by FWuellhorst — CHANGES_REQUESTED** [review]
Thanks for the PR! Prior to a functional check, some style changes :)

File: AixLib/ThermalZones/ReducedOrder/Multizone/BaseClasses/PartialMultizone.mo (hunk 1 above)
Comment (larissakuehn) [reply AixLib/ThermalZones/ReducedOrder/Multizone/BaseClasses/PartialMultizone.mo:24]: I changed description of use_NaturalAirExchgange to "Consider natural infiltration and ventilation by setting true" and of use_MechanicalAirExchange to "Consider mechanical ventilation by setting true" within ThermalZone and PartialMultizone

**Review by FWuellhorst — APPROVED** [review]
Everything looks good! Thanks for the nice work :)

**Pull request merged**